tokens/sys/breakpoint.json

```diff
@@ -0,0 +1,29 @@
+{
+  "breakpoints": {
+    "zero": {
+      "value": "0rem",
+      "type": "dimension",
+      "description": "Use to set a media query `min-width` below small."
+    },
+    "s": {
+      "value": "{base.unit} * 80",
```

Comment:
The zero value is in `rem`, but all the others are in `px`. All our token values for development will be in rem. I'm not sure if you'd like to convert that here, or we can do it in the transform step, but all the values should be the same unit.

Reply:
hm... we defined these as `rem` in tokens studio. for some reason it shows up as REM for me -- adding a screenshot for context.

![Canvas  System (Beta)-2024-02-28 cleanshot@2x](https://github.com/Workday/canvas-tokens-studio/assets/15917587/28dad624-14f7-418e-9630-a781f07c0b35)

Reply:
the multiplier `base.unit` should be `0.25rem`

Reply:
Ah, okay. My mistake. Looks like we do the same with system space tokens.